projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn-base.json

```
@@ -0,0 +1,211 @@
+    "gui.refurbished_furniture.workbench.search_neighbours.on": "包含相邻容器",
+    "gui.refurbished_furniture.experience_level": "%s/%s 级",
+    "gui.refurbished_furniture.experience_points": "%s 点",
+    "gui.refurbished_furniture.withdraw_experience": "提取经验",
+    "gui.refurbished_furniture.requires_power": "需要从%s供电",
+    "subtitle.refurbished_furniture.package_open": "纸箱：撕裂",
+    "subtitle.refurbished_furniture.chair_slide": "木头：滑动",
+    "subtitle.refurbished_furniture.doorbell_chime": "门铃：鸣响",
+    "subtitle.refurbished_furniture.electricity_generator_engine": "发电机",
+    "subtitle.refurbished_furniture.storage_jar_insert_item": "物品：移动",
+    "subtitle.refurbished_furniture.recycle_bin_engine": "碾压",
+    "subtitle.refurbished_furniture.ceiling_fan_spin": "风扇：旋转",
+    "subtitle.refurbished_furniture.lightswitch_flick": "开关：拨动",
+    "subtitle.refurbished_furniture.trampoline_bounce": "弹跳",
+    "subtitle.refurbished_furniture.trampoline_super_bounce": "强力弹跳",
+    "subtitle.refurbished_furniture.television_channel.colour_test": "高音调",
+    "subtitle.refurbished_furniture.television_channel.white_noise": "白噪音",
+    "subtitle.refurbished_furniture.television_channel.dance_music": "舞曲节拍",
+    "subtitle.refurbished_furniture.television_channel.villager_news": "新闻频道",
+    "subtitle.refurbished_furniture.television_channel.chirp_song": "鸟鸣为歌",
+    "subtitle.refurbished_furniture.television_channel.ocean_sunset": "LoFi节拍",
+    "subtitle.refurbished_furniture.television_channel.blocky_game": "钢琴音乐",
+    "subtitle.refurbished_furniture.television_channel.retro_song": "复古街机歌曲",
+    "subtitle.refurbished_furniture.frying_pan.place_ingredient": "啪嗒",
+    "subtitle.refurbished_furniture.frying_pan.break": "金属：撞击",
+    "subtitle.refurbished_furniture.frying_pan.hit": "金属：敲击",
+    "subtitle.refurbished_furniture.frying_pan.place": "金属：放置",
+    "subtitle.refurbished_furniture.frying_pan.step": "金属：脚步",
+    "subtitle.refurbished_furniture.frying_pan.sizzling": "嘶嘶",
+    "subtitle.refurbished_furniture.toaster.down": "烤面包机：启动",
+    "subtitle.refurbished_furniture.toaster.pop": "烤面包机：弹出",
+    "subtitle.refurbished_furniture.toaster.insert": "烤面包机：晃动",
+    "subtitle.refurbished_furniture.cooler.open": "冷藏箱：打开",
+    "subtitle.refurbished_furniture.cooler.close": "冷藏箱：关闭",
+    "subtitle.refurbished_furniture.microwave.open": "微波炉：打开",
+    "subtitle.refurbished_furniture.microwave.close": "微波炉：关闭",
+    "subtitle.refurbished_furniture.kitchen_drawer.open": "抽屉：滑动打开",
+    "subtitle.refurbished_furniture.kitchen_drawer.close": "抽屉：滑动关闭",
+    "subtitle.refurbished_furniture.cutting_board.place_ingredient": "物品：放置",
+    "subtitle.refurbished_furniture.kitchen_sink.fill": "水：流动",
+    "subtitle.refurbished_furniture.fridge.open": "冰箱：打开",
+    "subtitle.refurbished_furniture.fridge.close": "冰箱：关闭",
+    "subtitle.refurbished_furniture.stove.open": "炉灶：打开",
+    "subtitle.refurbished_furniture.stove.close": "炉灶：关闭",

```

```suggestion
    "subtitle.refurbished_furniture.stove.open": "烤箱：打开",
    "subtitle.refurbished_furniture.stove.close": "烤箱：关闭",
```
算是遵循原文，毕竟炉灶下方有个烤箱，也确实是打开或关上烤箱时的音效